civihr_default_theme/scss/components/_button.scss

```diff
@@ -2,7 +2,11 @@
   border: transparent;
   padding: 0 10px;
 
-  & + .dropdown-menu > li > a {
+  + .dropdown-menu a {
     color: $brand-primary;
   }
 }
+
+.btn-lg {
```

Comment:
You are overriding a basic bootstrap style, but I cannot find any explanation for why this was necessary in the PR. You are basically altering the look of a variation of a fundamental bootstrap component, and this must then be very well justified

Reply:
In Shoreditch the large button has the base font size, while in original Bootstrap the font is bigger for larger buttons. SSP should be consistent with Shoreditch in this particular point, I think, because there are elements reused in both Admin and SSP and they should look the same.

Shoreditch:
![image](https://user-images.githubusercontent.com/3973243/37768159-87d63388-2dc4-11e8-9a4b-54eaa746b962.png)

SSP before fix:
![image](https://user-images.githubusercontent.com/3973243/37768223-ba39d9f6-2dc4-11e8-967a-c0bd211b7fec.png)

SSP after fix:
![image](https://user-images.githubusercontent.com/3973243/37768282-f2a59168-2dc4-11e8-9c67-949f3b10a144.png)